deployer.go

```diff
@@ -349,6 +365,34 @@ func (d *deployer) performSequentialDeployment(sg serviceGroup) {
 			break
 		}
 
+		// read from etcd entry for app
+		//TODO context with timeout
+		etcdResp, err := d.etcdapi.Get(context.Background(), fmt.Sprintf("/ft/services/%s/healthcheck", getServiceName(u.Name)), nil)
+		if err != nil {
+			//TODO default to wait for unit
+			log.Printf("Error while getting etcd key for app from %s: %v", fmt.Sprintf("/ft/services/%s/healthcheck", getServiceName(u.Name)), err.Error())
+		}
+		log.Printf("HC node value: %s", etcdResp.Node.Value)
+		if etcdResp.Node.Value == "true" {
+			etcdResp, err := d.etcdapi.Get(context.Background(), "/ft/config/environment_tag", nil)
+			if err != nil {
+				//TODO default to wait for unit
+				log.Printf("Error while getting envrionment from %s: %v", "/ft/config/environment_tag", err.Error())
+			}
+			env := etcdResp.Node.Value
+			//TODO check gtg Endpoints
+			gtgResp, err := d.httpClient.Get(fmt.Sprintf("https://%s-up-coco.ft.com/__%s/__gtg", env, getServiceName(u.Name)))
```

Comment:
Since you're delving into etcd anyway, I wonder about the pros/cons of going directly to the services port to call the health check, instead of going via vulcan?
(which would also be robust against future changes to the __foo service pattern if we chose to change it.)

Also, what is the reason for going all the way out through the elb?